homeassistant/components/light/isy994.py

```diff
@@ -69,6 +69,10 @@ def turn_on(self, brightness=100, **kwargs) -> None:
             _LOGGER.debug('Unable to turn on light.')
 
     @property
+    def state_attributes(self):
```

Comment:
Thanks for your assistance. Truth be told: I don't know why this works or how it should be. I pulled this code out of an earlier build when the dimmer slider was working.

I don't do enough dev to understand your suggestion to fix the error.